fe1-web/res/adhoc-implementations/WalletCryptoLibrary.native.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * TODO: should return the crypto library for native context
+ */
+export class WalletCryptoLibrary {
+  public static getCrypto() : undefined {
+    return undefined;
+  }
+}
```

Comment:
why the `WalletCryptoLibrary` name?
why a class just for a single static method? 
why return undefined when we know the interface we want to respect?  

```suggestion
export  getCrypto(): Crypto {
    throw new Exception('not yet implemented');
}
```